### PR TITLE
Czech localization

### PR DESCRIPTION
**Pull request by VladWinner**

### CODE REVIEW

File: specops/localizedstrings/english.json [hunk 1]
```diff
@@ -314,7 +313,7 @@
     "SO_SNOWRACE1_CLIFFHANGER_GATES": "Gates hit",
     "SO_SNOWRACE1_CLIFFHANGER_OBJ_HARDENED": "Reach the Safe House in 2 minutes.",
     "SO_SNOWRACE1_CLIFFHANGER_OBJ_VETERAN": "Reach the Safe House in 1:30 minutes.",
-    "SO_SNOWRACE1_CLIFFHANGER_TIMESPLASH": "+&&1 Seconds",
+    "SO_SNOWRACE1_CLIFFHANGER_TIMESPLASH": "+&&1 seconds",
```
Comment (fedddddd) [inline specops/localizedstrings/english.json:316]: Leave it as "Seconds"

File: specops/localizedstrings/english.json [hunk 2]
```diff
@@ -28,7 +28,6 @@
     "MENU_SP_LEVEL_AVG_TIME": "Est. Completion Time: ",
     "MENU_SP_LEVEL_YOUR_BEST_TIME": "Your Best Time: ",
     "MENU_SP_TYPE_TIMED_DRIVING": "Timed Driving",
-    "MENU_SP_SPECIAL_OPS_CAPS": "SPECIAL OPS",
```
Comment (fedddddd) [inline specops/localizedstrings/english.json:31]: Why?

File: specops/ui_scripts/specops_ui/main.lua [hunk 3]
```diff
@@ -19,7 +19,7 @@ function frontend()
         end
     
         if (args[1] == "@MENU_SP_CAMPAIGN") then
-            return "SPECIAL OPS"
+            return "@MENU_SP_SPECIAL_OPS_CAPS"
```
Comment (fedddddd) [inline specops/ui_scripts/specops_ui/main.lua:22]: Call localize on that

**Review by fedddddd — CHANGES_REQUESTED** [review]

File: specops/localizedstrings/english.json (hunk 2 above)
Comment (VladWinner) [reply specops/localizedstrings/english.json:31]: This unnecessarily overwrites already translated string from built-in localizations.
182247050-1da979e1-90b5-4b50-b155-b6b66fda94a0 (1)

**Pull request merged**